src/Image.ts

```diff
@@ -437,7 +437,9 @@ export class Image {
     channel: number,
     value: number,
   ): void {
-    this.data[(row * this.width + column) * this.channels + channel] = value;
+    if (column >= 0 && column < this.width && row >= 0 && row < this.height) {
```

Comment:
We probably want to avoid adding checks here for performance reasons. This method is called very often, often in places where the call is already safe.

If we really need this, then I would rather do that in new method.

Reply:
I think the check actually doesn't do anything because assigning out of bounds on a Uint8Array or likewise is a noop.

![CleanShot 2024-09-18 at 14 17 10](https://github.com/user-attachments/assets/93b9ffa1-41a2-48e4-bde7-3d52aafd390f)